### DISCUSSION TITLE
DataGrid SelectionUnit

### DISCUSSION THREAD

**Discussion by KrzysztofDusko** [AvaloniaUI/Avalonia#8562] [Q&A] [answered]
There is any way to select few cells, like Selection Unit =  Cell mode ?
https://docs.microsoft.com/pl-pl/dotnet/api/system.windows.controls.datagrid.selectionunit?view=windowsdesktop-6.0

**Comment by timunie** [comment, marked as answer]
I think this is unfortunately not implemented and most likely it will never. The data grid is one of the most complex controls tbh. There will be a replacement for this control in the future, see: https://docs.avaloniaui.net/docs/controls/treedatagrid
